package/starling_daemon/overlay/etc/endpoint_router/sbp_router_starling.yml

```diff
@@ -0,0 +1,248 @@
+name: SBP_ROUTER
```

Comment:
Our router configurations are starting to balloon `n!` style with each new 'stream diverting' process we introduce, i.e. in order for this to truly work there would need to be an `sbp_router_startling_smoothpose.yml`. This a more general comment but I feel like we need to re-architect messaging in a way that moves towards a true pub-sub topic-per-msg-id streaming data model. @silverjam thoughts?

Reply:
I think we can add "stream diversion" as a requirement for the new router API.